chain/network/src/stats/rate_counter.rs

```diff
@@ -15,10 +15,13 @@
 use std::collections::VecDeque;
 use std::time::{Duration, SystemTime};
 
-const MINUTE_IN_MILLIS: u128 = 60_000;
+const MINUTE_IN_MILLIS: Duration = Duration::from_secs(60);
```

Comment:
I think I'd actually leave it as u128 -- as we only add it to the other integer (and I'm not sure how much calling '.to_millis' add to compute cost)

or - we could 'migrate' all the things to be a duration: expiration_timestamp and return values from millis_since_epoch

Reply:
Ok, I changed while `rate_counter.rs` to use `SystemTime`/`Duration`